pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -335,6 +337,13 @@ func Run() {
 	flowStatTimer := flextimer.NewRangeTicker(time.Duration(fmin),
 		time.Duration(fmax))
 
+	nexthopProbeIntv := time.Duration(NhProbeInteval * time.Second) // 15 sec
+	pmax := float64(nexthopProbeIntv)
+	pmin := pmax * 0.9
```

Comment:
This can result in self synchronization. Need to be 0.3.
See http://www.icir.org/floyd/papers/sync_94.pdf which says the random range should be between 0.5 and 1.5 which is the same as between 0.3 and 1.0.

Reply:
ok, didn't know that. i'll probably change those into timer vs ticker. reschedule one when current one is ending.